JS-Functions/js-functions-2_greeting-page/js/index.js

```diff
@@ -22,13 +22,40 @@ Change the contents of this page depending on the current day and time.
 
 const display = document.querySelector('[data-js="display"]');
 
+
 function getGreeting() {
-  // Code here
+  const timeNow = new Date().getHours();
+  console.log(timeNow)
+  if (timeNow >= 6 && timeNow <= 11) {
+    return "Good Morning"
+  }
+  else if (timeNow <= 17) {
+    return "Good Afternoon"
+  }
+  else if (timeNow <= 21) {
+    return "Good Evening"
+  } else {
```

Comment:
better to use else if (hours >= 23 || hours <= 5) { as it explains the logic behind it